lib/stamp/emitters/two_digits_only.rb

```diff
@@ -0,0 +1,20 @@
+module Stamp
+  module Emitters
+    # Emits the given field as a two-digit number with a leading
+    # zero if necessary.
+    class TwoDigitsOnly
+      attr_reader :field, :add_zero
+
+      # @param [field] the field to be formatted (e.g. +:year+)
+      def initialize(field, add_zero = false)
```

Comment:
And if that's the direction we go, couldn't this just be done by the TwoDigit emitter (including the `% 100`; it _is_ called "TwoDigit" after all)?